src/Util/UserManipulator.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\UserBundle\Util;
+
+use Sonata\UserBundle\Model\UserInterface;
+use Sonata\UserBundle\Model\UserManagerInterface;
+
+/**
+ * @internal
+ */
+final class UserManipulator
```

Comment:
Not sure if this class makes sense or it is better to use the UserManager we already have, thoughts @VincentLanglet ?

Reply:
Do we need all these methods ? You're only using `activate` in your command

Reply:
There are more commands coming, we will use all the methods.

Reply:
With the first command I would just write directly
```
$user = $this->userManager->findUserByUsername();

$user->setEnabled(true);

$this->userManager->save($user);
```
directly in the command.

Reply:
Done.